sdk/test/NetStandard/UnitTests/AWSSDK.UnitTests.Custom.NetStandard.csproj

```diff
@@ -44,12 +56,10 @@ This project file should not be used as part of a release pipeline.
 
     <ItemGroup>
         <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.7.2" />
-        <PackageReference Include="xunit.runner.visualstudio" Version="2.4.5">
```

Comment:
Was removing `PrivateAssets` intentional?

Reply:
Ah I forgot to put that back after some experimentation. I just updated it and added it back!